blockstore/src/indexed_db_blockstore.rs

```diff
@@ -0,0 +1,183 @@
+use cid::CidGeneric;
+use js_sys::Uint8Array;
+use rexie::{KeyRange, ObjectStore, Rexie, Store, TransactionMode};
+use send_wrapper::SendWrapper;
+use wasm_bindgen::{JsCast, JsValue};
+
+use crate::{convert_cid, Blockstore, BlockstoreError, Result};
+
+/// indexeddb version, needs to be incremented on every schema schange
+const DB_VERSION: u32 = 1;
+
+const BLOCKS_STORE: &str = "BLOCKSTORE.BLOCKS";
```

Comment:
```suggestion
const BLOCK_STORE: &str = "BLOCKSTORE.BLOCKS";
```

Reply:
ah this one was actually intentional :D a store for blocks. I can change that if you want

Reply:
🤔 wouldn't that still be singular? Like "flower shop" or "grain store"? fwiw google is calling their api for storing and retrieving blocks of data (for credential purposes) [Block Store](https://developers.google.com/identity/blockstore/android).
Where's native speaker when you need one 😿